cpp/src/lib/rdma_proxy_client.cc

```diff
@@ -41,64 +42,90 @@ std::string _build_msg(const std::string &prefix) {
 }
 
 void RDMAProxy_client::_really_write(const char *buf, const int len) {
+  if (len <= 0) {
+    return;
```

Comment:
Meta question as this is not something newly introduced: is `len < 0` something that the code should tolerate, or rather something to complain about via `assert` / exception, or to be ruled out altogether with an unsigned typed?